backend/app/core/yaml_init.py

```
@@ -656,6 +688,19 @@ def run_yaml_initialization(db: Session, skip_lock: bool = False) -> Dict[str, A
         logger.error(f"Failed to create default user: {e}", exc_info=True)
         return {"status": "error", "reason": "failed to create default user"}
 
+    # Ensure organization knowledge base exists
+    try:
+        logger.info("Ensuring organization_knowledge namespace exists...")
+        org_kb_created = ensure_organization_knowledge_namespace(db, user_id)
```

Ruff (0.14.14)

[error] Undefined name `ensure_organization_knowledge_namespace`

(F821)